mixins/rtl-mixin.js

```diff
@@ -4,6 +4,10 @@ export const RtlMixin = dedupeMixin(superclass => class extends superclass {
 
 	static get properties() {
 		return {
+			/**
+			 * The directionality of the text of the document
+			 * @type {'ltr'|'rtl'}
```

Comment:
Dir will actually not be set if the direction is not RTL, so I think the only valid option is 'rtl' or 'undefined'.

Reply:
Ah makes sense. Thoughts on if we should include that undefined would correspond to an ltr display then?

Reply:
I think it would default to the document `dir`.  In practice, consumers should never be setting this explicitly except in test cases.  We would never want someone to set this to a value (overriding the document `dir`) without realizing the effect.  I'd _almost_ ignore this, or add to the comment as such.

Reply:
I like where this ended up. Agreed that it should be ignored as Annabel said it's just a normal HTML property and our behaviour is basically the same as a native DOM node -- native nodes will have a value of empty string vs. ours will be undefined.